spec/ysh-proc.test.sh

```diff
@@ -451,3 +465,27 @@ p word (42, n=99) {
 (Int)   99
 Block
 ## END
+
+#### unset and procs
```

Comment:
Maybe clarify the description here?   Now regular `unset` can unset a proc, it doesn't have to be `unset -f`?

That seems right